examples/quill.html

```diff
@@ -164,6 +194,7 @@
               continue;
             }
 
+            const actorName = peers.get(actor).get('username');
```

Comment:
`peers.get(actor).get('username');` is not working!

```quill.html:198 Uncaught TypeError: peers.get(...).get is not a function
    at RichText.onChangesHandler (quill.html:198)
    at RichText.updateSelection (rich_text.ts:130)
    at SelectOperation.execute (select_operation.ts:37)
    at Change.execute (change.ts:45)
    at Document.applyChanges (document.ts:203)
    at Document.applyChangePack (document.ts:98)
    at client.ts:377
    at Array.<anonymous> (index.js:60)
    at yc (index.js:53)
    at S.<anonymous> (index.js:51)
    ```
    
`peers.get(actor)` does this by returning an object. It can be improved with `peers.get(actor).username`.

Reply:
Thank you. I removed `Map` from API exposed to users.